### PR TITLE
Rules UI updates

### PR DESCRIPTION
**Pull request by Stanton**
A collection of tweaks for the XFP implementation of the Pulsar Rules pattern.

* Fix issue preventing the proper negation of `show_or` & `show_and`
* Add utility classes to remove hover and border effects

- [x] Fix 'labels on top of inputs' issue
- [x] Fix '[odd borders in tablet mode](https://github.com/jadu/pulsar/pull/615#issuecomment-321762510)' issue
- [x] Fix '[Centered instructions page text](https://github.com/jadu/pulsar/pull/615#issuecomment-321762669)' issue
- [x] Fix '[left hand table border](https://github.com/jadu/pulsar/pull/615#issuecomment-321762824)' issue

----

## Displaying errors

You will normally need to do two things to properly indicate errors in a rule block.

1. Add a danger panel to the top of the UI, explaining there are issues that need to be resolved further down in the UI
1. Add an error message to the rule block
1. Highlight the specific field causing the error

rule-error-four

The `error` option, when supplied, will automatically apply the required error styling to the rule block.

### Adding error messages to the rule block

{% raw %}
```twig
{{
    rule.block__or({
        'error': 'Something went wrong...'
        ...
```
{% endraw %}

rule-error-one

In some cases you may need to add more than one error (e.g. field 1 has x error, field 2 has y error), the `error` field can accept an array of strings rather than a single string.

{% raw %}
```twig
{{
    rule.block__or({
        'error': [
            'This is error one',
            'This is error two'
        ]
        ...
```
{% endraw %}

rule-error-three

### Highlighting invalid fields

To properly communicate which specific form elements within a rule are causing the error, you should add the common `has-error` class where appropriate.

{% raw %}
```twig
{{
    rule.block__or({
        'error': 'Something went wrong...',
        'inputs': [
            form.select2({
                'class': 'has-error'
                ...
```
{% endraw %}

rule-error-two

### CODE REVIEW

**Comment by Stanton** [comment]
Odd borders when in tablet mode

screen shot 2017-08-11 at 09 58 46

**Comment by Stanton** [comment]
Centered instructions page text and misaligned toggle in small screen mode
screen shot 2017-08-11 at 09 59 21

**Comment by Stanton** [comment]
Left hand table border needs removing in tablet mode
screen shot 2017-08-11 at 10 00 19

File: stylesheets/_layout.settings.scss [hunk 1]
```diff
@@ -226,9 +216,21 @@
...
 
-.tab__settings .form__group--toggle .controls {
-    float: right;
-    width: 4em;
+.tab__settings .form__group--toggle {
+    > .control__label {
+        width: 100%;
+
+        @include respond-min($screen-small) {
+            padding-right: 0;
```
Comment (jamesjacobs) [inline stylesheets/_layout.settings.scss]: Alpha order, padding after margin

**Pull request merged**